yt/docs/en/_includes/admin-guide/identity-sync.md

```diff
@@ -0,0 +1,17 @@
+# Import users and groups from the external systems.
```

Comment:
1. Here, period is not needed at the end of the header.
2. Maybe `Importing users and groups from the external systems` or `Users and groups import from the external systems` ?